### ISSUE TITLE
Every time the same notification

### ISSUE THREAD

**Issue by johan12345** [AndlyticsProject/andlytics#665]
Every time the automatic sync runs and when I open the app, the same notification is shown that one of my apps has got new ratings and downloads, even if nothing has changed since the last sync.

**Comment by Brblol** [comment]
I had that happen to me for a while. I had to uninstall and install again

**Comment by johan12345** [comment]
That seems to have worked. But still, it is probably a bug that caused this to happen in the first place.

**Comment by willlunniss** [comment]
Notifications are only supposed to show if a change is detected. Do you have a very large number of comments?

**Comment by johan12345** [comment]
No, I have got four apps on two different accounts, and the number of comments and ratings per app is below 30 on all of them.

**Comment by willlunniss** [comment]
OK. Unfortunately because these sorts of problems are often dependent on the particular users apps you are going to have to build and install a debug build and have a look to see why it is happening. Make sure you export the data from the release app so that you can import it into the debug version.

Relevant bits of code include:

https://github.com/AndlyticsProject/andlytics/blob/6967dab5b5f86b83201b64bfd67478bc1326732d/src/com/github/andlyticsproject/Main.java#L499

https://github.com/AndlyticsProject/andlytics/blob/dev/src/com/github/andlyticsproject/sync/NotificationHandler.java#L62

**Comment by johan12345** [comment]
Because I reinstalled the app and the problem didn't occur again after that, I'll probably have to wait until it happens again before I do that.
